workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Injection/RequiredFieldInjector.cs

```diff
@@ -19,13 +19,15 @@ public class RequiredFieldInjector
             new Dictionary<Type, List<uint>>();
         private readonly Dictionary<Type, List<uint>> componentAuthRequirementsForBehaviours =
             new Dictionary<Type, List<uint>>();
+        private readonly Dictionary<Type, string[]> componentWorkerTypeRequirementsForBehaviours =
```

Comment:
Tempted to make a class wrapper for all these `componentTypeTo...` maps in this block

Reply:
I mean having one `dict<Type, SomeClass{int, int, string[]}>` instead of multiple maps of `Type,int`, `Type,int`, `Type,string[]`

Reply:
I'm personally indifferent about this, i.e. wither approach is fine.

Reply:
UTY-1239